build/compile.php

```diff
@@ -51,6 +51,10 @@ function remove_header($contents)
 $files = new RecursiveIteratorIterator(new RecursiveDirectoryIterator(SP_PATH . '/library/SimplePie'));
```

Comment:
You can just pass `FilesystemIterator::SKIP_DOTS` to [`RecursiveDirectoryIterator`](https://www.php.net/manual/en/class.recursiveiteratoriterator.php).